### PR TITLE
Auxiliary branch test feature

### PR DESCRIPTION
**Pull request by chiztechnology**
I just set up some features on my new branch !
please check them and let me know if we can add them to the main repository
Kindest regards
😃

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @chiztechnology, 
# Status: Changes Required ♻️

Hearty cheers for making it into Microverse🎉.

You have done a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

a little more


## Good Highlights✔

- Linter checks are passing
- Descriptive PR
- Adherence to GitHub flow
- Test file properly ignored


## Required Changes ♻️

- Your README file looks great. Kindly consider making it more descriptive by adding as many sections as possible from the [README template](https://github.com/microverseinc/readme-template) provided by Microverse and remember to customize it.
- In future projects, kindly let  the commit messages be written in the imperative tense and capitalized. This is a recommended practice. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). 
Here are some examples of descriptive commits:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chiztechnology** [comment]
I've just updated the readme file as requested.
Thank you for your review.
I'll better write my commits now following the rules learned on Microverse.
✅

**Comment by emmiiorji** [comment]
Hi @chiztechnology,

Sounds good. Keep up the good work.

You may go ahead and resubmit the project on your dashboard and the next reviewer will do justice.

Happy coding 👏👏.

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @chiztechnology, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 💪 
- No linter errors
- correct flow is used
- good improvement based on the previous reviewer's suggestions
- good PR


 ## Required Changes ♻️

- [ ] It looks like you have not updated the README file on this current branch that you are working on. Take a look at the following screenshot. You will find out that there are few content in the README 👇 
image
Please as requested by the previous reviewer, update your README file and Remember to customize it to fit your project specification 😃 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chiztechnology** [comment]
thank you for your review, I unfortunatelly committed the changes in the main branch in stead of the other auxiliary branch.
I've corrected so !
Thanks alot
I really appreciate

File: README.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+
+The project includes :
+
 1. Basic HTML page
 2. some CSS code
+
+# Built With
+HTML
+CSS
+
```
Comment (bienvenuushindi) [inline README.md:12]: **[OPTIONAL]**
You have done well :clap:  adding most of the readme sections :ok_hand:. You have one more section (Getting Started) which is not there yet. As you can see from [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) This section includes all the necessary steps to clone your GitHub repository. Please make sure you include this section and all the necessary steps. This section should be after the Built With as it is in the template
Your section should be similar to something like this :arrow_down_small: 

## Getting Started
 Please follow the instructions to clone my repo [You can put  your own text]

To get a local copy, follow these simple example steps.

Clone this repository or download the Zip folder:

`git clone <add the link to your repo  here>`

Navigate to the location of the folder in your machine:

`you@your-Pc-name:~$ cd <folder>
`
    Press Enter to navigate to your local clone.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+
+# Author
+👤 Author
+
+GitHub: @chiztechnology
+
+Twitter: @IsraelChizungu
+
+LinkedIn: IsraelChizungu
```
Comment (bienvenuushindi) [inline README.md:21]: **[OPTIONAL]**
Please kindly use this format :point_down: to add  your different accounts and their respective links
- Account: [Name](add profile link here ) This is the markdown to display a link 
### An updated version of your accounts should be similar to this :point_down: 
- GitHub: [@chiztechnology](https://github.com/chiztechnology/)
- Twitter: [@IsraelChizungu](<add here your twitter profile link>)
- LinkedIn: [@IsraelChizungu](<add here your linkedin profile link>)

**Review by bienvenuushindi — APPROVED** [review]
## STATUS APPROVED :green_circle: 
Hi @chiztechnology,

Your project is complete! Great job for making all the required changes :1st_place_medal:. There is nothing else to say other than... it's time to merge it :champagne:  :shipit: . Congratulations! :tada::100::star2:
Congratulations! 🎉 

appr
# HIGHLIGHTS :+1: 
  - GitHub flow is used correctly.  :heavy_check_mark:
  - All linter checks are ok.  :heavy_check_mark:
  - Good commit history :heavy_check_mark: 
## Optional suggestions
 - [x] Kindly check comments under the review.

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by chiztechnology** [comment]
thanks for all your comments ! they will help me to improve my skills and to keep doing better ! I really appreciate

**Pull request merged**